### PR TITLE
feat: add progress alert

### PR DESCRIPTION
**Pull request by ahmaxed**
Checklist:

here is the progress bar in action:

progress

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: client/src/templates/Introduction/intro.css [hunk 1]
```diff
@@ -449,6 +449,57 @@ ToDo: find out why, and remove the need for it */
...
+  color: var(--blue70);
+}
+
+.annual-donation-alert .progress-bar-wrap,
+.progress-bar-background {
+  background-color: transparent;
+}
+
+.annual-donation-alert hr {
+  border-top-color: var(--blue70);
```
Comment (ahmaxed) [inline client/src/templates/Introduction/intro.css:470]:
```suggestion
  border-top-color: var(--blue10);
```

**Review by Nirajn2311 — APPROVED** [review]
LGTM 👍 

Some things which would be good to have if possible:
* Why not make the year a dynamic value so we can reuse the alert box without having to remember to update the year
* The progress could fill up from 0 on load.

**Comment by ahmaxed** [comment]
@moT01, that sounds like a good suggestion. Should we get this in and iterate on it?

**Review by moT01 — COMMENTED** [review]
I would be fine getting this in. I am seeing a different color for the HR (in both light and dark mode) - which makes it hard to see:

Screenshot 2023-12-06 at 12 42 45 PM

If you're fine with that, we can go with it like this for now @ahmaxed.

**Review by moT01 — APPROVED** [review]

**Pull request merged**